src/uci.cpp

```
@@ -0,0 +1,422 @@
+    return this->optionType;
+}
+
+std::string UCIOption::getName() {
+    return this->name;
+}
+
+std::string UCIOption::getValue() {
+    return this->value;
+}
+
+void UCIOption::setValue(std::string value) {
```

cppcheck

[performance] Function parameter 'value' should be passed by const reference.

(passedByValue)